src/index.js

```diff
@@ -0,0 +1,74 @@
+import 'lodash';
+import './style.css';
+import sync from './assets/icons/sync.png';
+import downLeft from './assets/icons/down_left.png';
+import menuIcon from './assets/icons/menuIcon.png';
+
+const todoContaiter = document.querySelector('.todoContaiter');
+const titleContainer = document.querySelector('.title-list');
+const descriptionContainer = document.querySelector('.description-list');
+
+const tasksArray = [
+  {
+    index: '3',
+    description: 'first task',
+    completed: false,
+  },
+  {
+    index: '1',
+    description: 'second task',
+    completed: false,
+  },
+  {
+    index: '2',
+    description: 'third task',
+    completed: false,
+  },
+];
+
+// sort by index
+tasksArray.sort((a, b) => a.index - b.index);
```

Comment:
- [ ] Good job on implementing the changes requested by the previous reviewer. However, the index is still a string that wraps a number, and as asked by the previous reviewer and in the requirement, the index should be a number so kindly fix this.
### Hint: remove the `' '` (quotes) that wrap the number, so it will be like:
```
const tasksArray = [
  {
    index: 3,
    description: 'first task',
    completed: false,
  },
  {
    index: 1,
    description: 'second task',
    completed: false,
  },
  {
    index: 2,
    description: 'third task',
    completed: false,
  },
];
```